### ISSUE TITLE
`update --all` downgrades packages, conda reverts the changes.

### ISSUE THREAD

**Issue by randolf-scholz** [mamba-org/mamba#1260]
Not sure if this is something package specific, but when I do `mamba update --all` with my current environment then I get:

```
  Package        Version  Build              Channel                    Size
──────────────────────────────────────────────────────────────────────────────
  Change:
──────────────────────────────────────────────────────────────────────────────

  - entrypoints      0.3  pyhd8ed1ab_1003    installed                      
  + entrypoints      0.3  py39hde42818_1002  conda-forge/linux-64     Cached
  - parquet-cpp    1.5.1  2                  installed                      
  + parquet-cpp    1.5.1  1                  conda-forge/linux-64     Cached
  - pickleshare    0.7.5  py_1003            installed                      
  + pickleshare    0.7.5  py39hde42818_1002  conda-forge/linux-64     Cached
  - tomlkit        0.7.2  pyha770c72_1       installed                      
  + tomlkit        0.7.2  py39hf3d152e_0     conda-forge/linux-64     Cached

  Summary:

  Change: 4 packages

  Total download: 0  B

──────────────────────────────────────────────────────────────────────────────
```

But if I subsequently use `conda update --all` it reverts these changes. The behaviour can be reproduced by

```ba
conda create --name example -c conda-forge entrypoints mamba
conda activate example
conda config --env --remove channels defaults
conda config --env --add channels conda-forge
conda config --env --show channels
conda update --all
mamba update --all
conda update --all
```

**Comment by wolfv** [comment]
Yes, that is a bug where we currently prefer the "arch"-version of a package. We need to fix this, thanks for the report!

**Comment by wolfv** [comment]
this was fixed with #1312
